src/main/java/org/logstash/dissect/search/DelimiterSearchStrategy.java

```diff
@@ -0,0 +1,5 @@
+package org.logstash.dissect.search;
+
+public interface DelimiterSearchStrategy {
```

Comment:
re: multibyte, I have one JUnit and one Ruby test that uses multibyte utf-8 strings - both pass.